code/src/components/TaskContent.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import { Header } from "./Header";
+import { ItemAdded } from "./ItemAdded";
+import { ItemList } from "./ItemList";
+import { ItemRemoved } from "./ItemRemoved";
+
+import styled from "styled-components";
+
+export const TaskContent = () => {
```

Comment:
Nice that you added this component to add structure to the page, makes it really clear!